src/test/ui/suggestions/type-ascription-instead-of-statement-end.stderr

```diff
@@ -0,0 +1,13 @@
+error: expected type, found `0`
+  --> $DIR/type-ascription-instead-of-statement-end.rs:15:5
+   |
+14 |     println!("test"):
+   |                     -
+   |                     |
+   |                     help: did you mean to end the statement here instead? `;`
```

Comment:
this sort of looks funny, perhaps because of the `^` having no label. I think I'd prefer:

```
error: expected type, found `0`
  --> $DIR/type-ascription-instead-of-statement-end.rs:15:5
   |
14 |     println!("test"):
   |                     -
   |                     |
   |                     help: did you mean to use a `;` here?
15 |     0;
   |     ^ expecting a type here because of type ascription

error: aborting due to previous error
```